File: PHP/Gym exercise/Gym/Plan.php
<?php

class Plan {
    private string $name;
    private int $price;
    private string $services;

    public function __construct(string $name, int $price, string $services) {
        $this->name = $name;
        $this->price = $price;
        $this->services = $services;
    }

    public function getName(): string {
        return $this->name;
    }

    public function getPrice(): int {
        return $this->price;
    }

    public function getServices(): String {
        return $this->services;
    }

  
    
}

File: JAVA/Gym exercise/com/gym/Plan.java
package com.gym;

public class Plan{
    private String name;
    private double price;
    private String services;


    public Plan (String name, double price, String services) {
        this.name = name;
        this.price = price;
        this.services = services;
    }

    public String getName() {
        return this.name;
    }

    public double getPrice() {
        return this.price;
    }

    public String getServices() {
        return this.services;
    }
}

